tests/appsec/suitespec.yml

```diff
@@ -84,8 +84,11 @@ suites:
       - '@remoteconfig'
       - tests/appsec/iast/*
       - tests/appsec/iast_packages/*
-    runner: riot
-    snapshot: true
+    retry: 2
+    runner: hatch
+    timeout: 50m
```

Comment:
nit: Should we bump this to 60? I saw that [one of the jobs failed due to timeout at 50mins](https://gitlab.ddbuild.io/DataDog/apm-reliability/dd-trace-py/-/jobs/748836447) on a previous commit.

Reply:
Thank you @erikayasuda for your suggestion and for catching that issue!! That branch failed after 50 minutes because I was testing this test suite on Python 3.13, and package installation was failing at every step. This indicates that the test suite isn't ready for Python 3.13 yet, so I removed it in the next commit and everything should work now ;)